content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -48,7 +48,7 @@ different purposes:
   kinds of Kubernetes data into a Pod
 - [CSI ephemeral volumes](#csi-ephemeral-volumes):
   similar to the previous volume kinds, but provided by special
-  [CSI drivers](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+  [CSI drivers](https://kubernetes-csi.github.io/docs/drivers.html)
```

Comment:
This change is not needed.  This doc is already referenced on line 52.

Reply:
Instead, you can update the URL on line 52 with https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html.

Reply:
@xing-yang  opened PR